material-overrides/404.html

```diff
@@ -1,12 +1,477 @@
+
+    /* Media queries and accessibility */
+    @media (max-width: 768px) {
+      .page-404-container { padding: 1rem; }
+      .polkadot-logo { width: 100px; height: 100px; }
+      .dot { width: 26px; height: 13px; }
+    }
+    @media (prefers-reduced-motion: reduce) {
+      .polkadot-logo, .dot, .bg-circle, .error-code, .error-title, .error-description, 
+      .dissolving-text, .dissolving-text::before {
+        animation: none; opacity: 1; transform: none;
+      }
+    }
+    .polkadot-logo:focus { outline: 2px solid #E6007A; outline-offset: 2px; }
+  </style>
+{% endblock %}
+
+
+{#
+  This block replaces the theme's main content area with your HTML.
+  Note that this will appear within the site's overall layout (header, nav, etc.).
+#}
 {% block content %}
-  <h1 class="not-found">404 - Not found</h1>
+  <div class="page-404-container">
+    <div class="background-decoration">
+        <div class="bg-circle"></div>
+        <div class="bg-circle"></div>
+        <div class="bg-circle"></div>
+        <div class="morphing-particle"></div>
+        <div class="morphing-particle"></div>
+        <div class="morphing-particle"></div>
+        <div class="morphing-particle"></div>
+        <div class="morphing-particle"></div>
+    </div>
+    
+    <div class="page-404-content">
+        <div class="polkadot-logo">
+            <div class="dot"></div>
+            <div class="dot"></div>
+            <div class="dot"></div>
+            <div class="dot"></div>
+            <div class="dot"></div>
+            <div class="dot"></div>
+        </div>
+        
+        <div class="error-code">404</div>
+        <h1 class="error-title">Page Not Found</h1>
+        <p class="error-description">
+            <span class="dissolving-text">The page you're looking for doesn't exist or has been moved.</span>
+            <a href="{{ config.site_url }}">Return to the Polkadot Docs</a>.
```

Comment:
Whenever I add styles, I try to align it with the brand and what already exists. So for this, I would use a button like they do on polkadot.com

Example: https://polkadot.com/blog/interview-with-daniel-cukier-jam-implementer/a/

<img width="1728" height="957" alt="Screenshot 2025-08-04 at 3 06 06 PM" src="https://github.com/user-attachments/assets/7cee5ed0-7a58-4589-abf1-cbf80b38bbe6" />